README.md

```diff
@@ -1,12 +1,12 @@
 
 # FlexVolume
-See [flexvolume/windows](flexvolume/windows) for kubernetes flexvolume plugins that can mount iSCSI and SMB on windows.
+See [flexvolume/windows](flexvolume/windows) for Kubernetes flexvolume plugins that can mount iSCSI and SMB on Windows Server.
```

Comment:
The plugins are not server specific and will run on client, I deliberately did not call out windows server.